config/default.yml

```diff
@@ -102,6 +102,13 @@ linters:
     enabled: true
     style: space
 
+  StrictLocals:
+    enabled: true
```

Comment:
I would suggest disabling this by default, since it seems like a relatively invasive change for existing codebases.